src/elements/content-sidebar/__tests__/SidebarNavTablist-test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import SidebarNavTablist from '../SidebarNavTablist';
+import {
+    SIDEBAR_VIEW_SKILLS,
+    SIDEBAR_VIEW_ACTIVITY,
+    SIDEBAR_VIEW_DETAILS,
+    SIDEBAR_VIEW_METADATA,
+    KEYS,
+} from '../../../constants';
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    withRouter: Component => Component,
+}));
+
+describe('elements/content-sidebar/SidebarNavTablist', () => {
+    test('should correctly render children', () => {
+        const MockChildren = () => <div />;
+
+        const wrapper = shallow(
+            <SidebarNavTablist>
+                <MockChildren />
+            </SidebarNavTablist>,
+        );
+
+        expect(wrapper.type()).toEqual('div');
+        expect(wrapper.hasClass('bcs-SidebarNav-main')).toBe(true);
+        expect(wrapper.find(MockChildren).length).toEqual(1);
+    });
+
+    describe('handleKeyDown', () => {
+        const viewList = [SIDEBAR_VIEW_ACTIVITY, SIDEBAR_VIEW_DETAILS, SIDEBAR_VIEW_SKILLS, SIDEBAR_VIEW_METADATA];
+
+        test('should navigate to previous tab', () => {
+            const history = createMemoryHistory();
+            history.push(`/${SIDEBAR_VIEW_DETAILS}`);
```

Comment:
You should be able to pass `initialEntries` as an option to `createMemoryHistory` to avoid needing to push here.